lib/vcs.ml

```diff
@@ -64,27 +64,24 @@ let find_git () =
       | dir, (_, `Exited 0) -> Ok (Some (v `Git git ~dir:Fpath.(v dir)))
       | _ -> Ok None )
 
-let err_git_exit cmd c = R.error_msgf "%a exited with code %d" Cmd.dump cmd c
-
-let err_git_signal cmd c =
-  R.error_msgf "%a exited with signal %d" Cmd.dump cmd c
+let cmd_error cmd = function
+  | `Exited c -> R.error_msgf "%a exited with code %d" Cmd.pp cmd c
```

Comment:
I wonder if we should spread this message across several lines so that it's more readable, something like:
```
Command:
git whatever --option val ...
exited with code x
```

What do you think?

Reply:
Or maybe:
```
The following command exited with code x:
git whatever --bla bli ...
```

Reply:
I like the last one better indeed, I will change that.